### ISSUE TITLE
Altibox's TV streams need a UA override

### ISSUE THREAD

**Issue by THEtomaso** [MoonchildProductions/UXP#507]
Altibox's Silverlight streams won't play when using Pale Moon's default UA mode (Firefox Compatibility).
I've contacted Altibox Norway several times, explaining the problem, but they don't even care to reply.
Since Altibox is one of Norway's and Denmark's largest broadband providers, providing the necessary UA overrides in Pale Moon by default, seems like a sensible solution.
The streams play just fine, using PM's "Native" UA mode.

Affected sites:
http://tvoveralt.altibox.dk/
http://tvoveralt.altibox.no/

(Pale Moon v28.0.0a4,x64,2018-06-14 / Win8.1Pro,x64)

Reference:
https://github.com/MoonchildProductions/Pale-Moon/issues/1403/

**Comment by mattatobin** [comment]
You have been told. Use the forum to discuss ua overrides. If they are important enough then they will be added but I am sick of your noise.

**Comment by THEtomaso** [comment]
I'm experiencing problems at altibox.no, using Pale Moon v28 (including v28.0.1,x64).
After a while, the Silverlight streams will switch from HD to SD, the picture becomes choppy, and I have to restart Pale Moon to fix it.

Don't know if this is another UA issue.
I tried modifying the override which is included in v28, making it identical to the one that I used with v27.*, but it didn't seem to make things any better.

For now, I went back to using Pale Moon v27.9.4,x64 for this particular service.
But I will run some further tests with v28, to see if I can find out more about what's going on.
I'll report back here with my findings.
